brave-clean-slate.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This script removes Brave browser configuration, bookmarks, extensions and residual files.
+# Usage: ./brave-clean-slate.sh
+
+sudo rm -rf /Library/Application\ Support/BraveSoftware
+rm -rf ~/Library/Application\ Support/BraveSoftware
+
+# Residual files
+rm -rf /Library/Saved Application State/com.brave.Browser.savedState
+rm -rf /Library/Caches/BraveSoftware
+rm -rf /Library/Preferences/com.brave.Browser.plist
```

Comment:
I feel the "Residual files" should be deleted as part of a normal invocation of `uninstall-brave.sh`. In other words, I would add the above three lines to `uninstall-brave.sh`.

Reply:
It would be nice to also remove `com.brave.Browser.beta.plist` and `com.brave.Browser.nightly.plist`.